src/parser/smt2/smt2_state.cpp

```diff
@@ -284,6 +293,7 @@ void Smt2State::addCoreSymbols()
 {
   defineType("Bool", d_solver->getBooleanSort(), true);
   Sort tupleSort = d_solver->mkTupleSort({});
+  defineType("Nullable", d_solver->mkNullableSort(tupleSort), true);
```

Comment:
This shouldn't be necessary. Can you try removing this line?

Reply:
Done.